build/_build/PrepareRelease/SetAllVersions.cs

```diff
@@ -140,15 +132,6 @@ public void Run()
                 "src/Datadog.Trace/TracerConstants.cs",
                 FourPartVersionReplace);
 
-            // Locked AssemblyVersion #.0.0.0 updates
```

Comment:
For the long-term, do you think we should keep shipping a shim for Datadog.Trace.AspNet when Datadog.Trace goes 2.0? Or just stop producing it once we hit 2.0? If we want to keep producing it, then we'll need to restore this

Reply:
Restored in 3911e194524affdf86edb87aeaa507e8030feb4f for now. We can decide later if we want to drop `Datadog.Trace.AspNet` in 2.0.

Reply:
Actually, if the reason to keep this is backwards-compat, should it stay at `1.0.0.0`? Otherwise users will need to update their `web.config` to specify `2.0.0.0` after the bump.

Reply:
Yeah you're absolutely right, we need to keep it at `1.0.0.0` always